members/urls.py

```diff
@@ -20,6 +20,7 @@
 
     path('', include('django.contrib.auth.urls')),
 
+    path('o/', include('oauth2_provider.urls', namespace='oauth2_provider')),
```

Comment:
This exposes the oauth URLs as `/member/o/...`. You probably wanted something like this, so they are available at `/o/...`?

```diff
--- a/mos/urls.py
+++ b/mos/urls.py
@@ -17,6 +17,7 @@ urlpatterns = [
 
     path('feeds/events/', EventFeed()),
 
+    path('o/', include('oauth2_provider.urls', namespace='oauth2_provider')),
     path('calendar/', include('cal.urls')),
     path('project/', include('projects.urls')),
     path('member/', include('members.urls')),
```